### DISCUSSION TITLE
Mirth Connect Public Comment

### DISCUSSION THREAD

**Discussion by CyRix76** [nextgenhealthcare/connect#5681] [Q&A] [answered]
Good morning,
I don't know if I'm in the right place but I have a question for you, I have installed mirth connect 3.5.2 on a production server, if I can't update the product for internal company reasons, can you tell me if patches are released for this version or is it mandatory to upgrade to latest versions?

Best regards
Riccardo

**Comment by pacmano1** [comment, marked as answer]
Patches are not released for older versions. Effectively you must upgrade.

**Comment by pladesma** [comment]
Correct, we don't release patches for old versions of Mirth Connect. We do have plans for an LTS version, but details about that are still to come.

**Reply by pacmano1** [reply]
Moved this to discussions where it should have been posted.
